bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/SearchPattern.java

```diff
@@ -72,7 +72,6 @@ public class SearchPattern {
 	 * <code>NullPointerException</code> type.</li>
 	 * </ul>
 	 *
-	 *
 	 * <br>
 	 * Can be combined with {@link #RULE_PREFIX_MATCH} match rule. For example, when
```

Comment:
> `Can be combined with {@link #RULE_PREFIX_MATCH} match rule.`

I believe this text is also problematic. Because in reality, there is no need to combine with the mentioned flag at all - the fallback is simply automatic. I'm not sure how to improve this easily, maybe we can leave this as-is for now...?

Reply:
Sounds fine for me.